.pipelines/templates/PackageBuildPRCheck.yml

```diff
@@ -10,6 +10,13 @@ parameters:
     default: ""
     displayName: "Daily build ID"
 
+  - name: isDailyBuildUpdateManifests
```

Comment:
On this level, I think we don't need to be so detailed on the other hand. What we care about during PR checks, if we should use the published public RPMs or a daily build ID, so I think we can simplify this to one parameter. The caller shouldn't need to worry about implementation details like the manifest updates.